message.go

```diff
@@ -36,6 +36,58 @@ const (
 )
 
 type Props map[string]string
+
+func (p Props) GetIntValue(key string) (int64, bool, error) {
+	val, ok := p[key]
+	if !ok {
+		return 0, false, fmt.Errorf("property %s not found", key)
```

Comment:
If you introduced bool for presence check, do not return error here. Just return 0, false, nil.